platforms/software/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishPomCustomizationIntegTest.groovy

```diff
@@ -256,7 +256,7 @@ class MavenPublishPomCustomizationIntegTest extends AbstractMavenPublishIntegTes
 
             model {
                 tasks.generatePomFileForEmptyMavenPublication {
-                    destination = 'build/generated-pom.xml'
+                    destination.set(file('build/generated-pom.xml'))
```

Comment:
❓ Can we use `destination = file('build/generated-pom.xml')` here or it doesn't work?